tests/bcl-test/System.IO.Compression.FileSystem/Info-mac.plist

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDisplayName</key>
+	<string>corelibtests</string>
```

Comment:
It's minor, but it's a little bit less confusing when looking at installed apps if the name matches the actual tests :smile: IOW maybe use something like `System.IO.Compression.FileSystemTests` as the `CFBundleDisplayName`.

Reply:
Sure, makes sense. Will update and merge the branch.